ocean/transects/compute_transects.py

```diff
@@ -99,44 +100,53 @@ def compute_transport(timeavg, mesh, mask, name='Drake Passage'):
       transport[i,j] = (dvEdge[start:stop,np.newaxis]*h[np.newaxis,:]*vel[start:stop,:] \
           *edgeSigns[j,start:stop,np.newaxis]).sum()*m3ps_to_Sv
 
-    print fname
- 
-  for j in range(nTransects):
+# Define some dictionaries for transect plotting
+  obsDict = {'Drake Passage':[120,175],'Tasmania-Ant':[147,167],'Africa-Ant':None,'Antilles Inflow':[-23.1,-13.7], \
+          'Mona Passage':[-3.8,-1.4],'Windward Passage':[-7.2,-6.8],'Florida-Cuba':[30,33],'Florida-Bahamas':[30,33], \
+          'Indonesian Throughflow':[-21,-11],'Agulhas':[-90,-50],'Mozambique Channel':[-20,-8], \
+          'Bering Strait':[0.17,1.49],'Lancaster Sound':None,'Fram Strait':None,'Robeson Channel':None}
```

Comment:
I accidentally doubled this work today, but this is a good check of both our methods.  The results are a little different.  Do you recall what references you used?  Thanks!

```
# observations (Positive values are north and eastward, see Ringler et al 2013)
obs = {'Drake Passage': {'mean': 134 , 'std dev': 14, 'ref': 'Nowlin and Klinck (1986)'}, 
       'Tasmania-Ant': {'mean': 157, 'std dev': 10, 'ref': 'Ganachaud and Wunsch (2000)'},
       'Africa-Ant': {'mean': 150, 'std dev': 30, 'ref': 'Estimated from Drake and Tasmania-Ant passages from Ganachaud and Wunsch (2000) http://onlinelibrary.wiley.com/doi/10.1029/2002JC001565/full'},
       'Indonesian Throughflow': {'mean': -15, 'std dev': 4, 'ref': 'Sprintall et al. (2009)'},
       'Agulhas': {'mean': -70, 'std dev': 20, 'ref': 'unclear from Ringler et al (2013)' },
       'Mozambique Channel': {'mean': -16, 'std dev': 13, 'ref': ' van der Werf et al. (2010)'},
       'Antilles Inflow': {'mean': -18.4, 'std dev': 4.7, 'ref': 'Johns et al. (2002) and Roemmich (1981)'},
       'Mona Passage': {'mean': -2.6, 'std dev': 1.2, 'ref': 'Johns et al. (2002) and Roemmich (1981).'},
       'Windward Passage': {'mean': -7.0, 'std dev': np.nan, 'ref': 'Johns et al. (2002) and Roemmich (1981).'},
       'Florida-Cuba': {'mean': 31, 'std dev': 1.5, 'ref': 'Johns et al. (2002) and Roemmich (1981).'},
       'Florida-Bahamas': {'mean': 31.5, 'std dev': 1.5, 'ref': 'Johns et al. (2002) and Roemmich (1981).'},
       'Bering Strait': {'mean': 0.8, 'std dev': 0.3, 'ref': 'Roach et al (1995) http://onlinelibrary.wiley.com/doi/10.1029/95JC01673/full'},
       'Lancaster Sound': {'mean': 0.8 , 'std dev': 0.3, 'ref': 'Prisenberg and Hamilton (2005) http://www.tandfonline.com/doi/abs/10.3137/ao.430101'},
       'Fram Strait': {'mean': -3, 'std dev': 3, 'ref': 'Schauer et al (2004) http://onlinelibrary.wiley.com/doi/10.1029/2003JC001823/full'},       
       'Robeson Channel': {'mean': -0.7 , 'std dev': .2 , 'ref': 'Maltrud and McClean (2005) https://www.sciencedirect.com/science/article/pii/S1463500303000684#BIB16'}
       }
```

Reply:
most of these values are from a table from @mark-petersen.  The Drake passage is altered based on new observations in a paper Mat M. sent around that showed the drake passage transport is 175 Sv.